README.md

```diff
@@ -1,3 +1,5 @@
+# This repository is no longer maintained, please visit https://github.com/unikraft/catalog/tree/main/library/nginx/1.15 
```

Comment:
Do not use heading formatting (i.e. starting with `#`). Use bold for the sentence. Link it like `please visit the [application catalog](https://github.com/unikraft/catalog/tree/main/library/nginx/1.15)`.